ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -187,24 +187,58 @@ public void after() {
   }
 
   @Test
-  public void testPrintGenericStreamedRow() {
+  public void testPrintStreamRow() {
     // Given:
-    final StreamedRow row = StreamedRow.row(genericRow("col_1", "col_2"));
+    final StreamedRow row = StreamedRow.streamRow(genericRow("col_1", "col_2"));
 
     // When:
     console.printStreamedRow(row);
 
     // Then:
+    assertThat(terminal.getOutputString(), containsString("col_1"));
+    assertThat(terminal.getOutputString(), containsString("col_2"));
+  }
+
+  @Test
+  public void testPrintTableRow() {
+    // Given:
+    final StreamedRow row = StreamedRow.tableRow(ImmutableList.of("k_0"), genericRow("col_1", "col_2"));
+
+    // When:
+    console.printStreamedRow(row);
+
+    // Then:
+    if (console.getOutputFormat() != OutputFormat.TABULAR) {
+      assertThat(terminal.getOutputString(), containsString("k_0"));
+    }
+    assertThat(terminal.getOutputString(), containsString("col_1"));
+    assertThat(terminal.getOutputString(), containsString("col_2"));
+  }
+
+  @Test
+  public void testPrintTableTombstone() {
+    // Given:
+    console.printStreamedRow(StreamedRow.pushHeader(new QueryId("id"), SCHEMA.key(), SCHEMA.value()));
+
+    final StreamedRow row = StreamedRow.tombstone(ImmutableList.of("k_0"));
+
+    // When:
+    console.printStreamedRow(row);
+
+    // Then:
+    assertThat(terminal.getOutputString(), containsString("k_0"));
+
     if (console.getOutputFormat() == OutputFormat.TABULAR) {
-      assertThat(terminal.getOutputString(), containsString("col_1"));
-      assertThat(terminal.getOutputString(), containsString("col_2"));
+
```

Comment:
Looks like something was meant to be here but was forgotten. Perhaps check that `<TOMBSTONE>` is present?